book/src/specs/kimchi.md

```diff
@@ -1297,7 +1297,7 @@ pub struct ProofEvaluations<Field> {
     /// evaluation of the poseidon selector polynomial
     pub poseidon_selector: Field,
     /// evaluation of the cairo selector polynomial
-    pub cairo_selector: [Field; crate::circuits::polynomials::turshi::CIRCUIT_GATE_COUNT],
+    pub cairo_selector: Option<[Field; crate::circuits::polynomials::turshi::CIRCUIT_GATE_COUNT]>,
```

Comment:
there's still this path that could be imported :o

Reply:
oh there! your eyes are well trained 👀